tests/amazon_test.sh

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env bash
 
 script_path="../main.nf"
-if [ -z $1]
+if [ -z $1] 
 then
     echo "No argument given, going to try to run ../main.nf"
 else
     script_path=$1
 fi
 
-data_path="/tmp"
+data_path="./test_data"
 if [ -d "./test_data" ]
```

Comment:
You can remove this `if` statement - all it does is set `data_path` to `./test_data` if `./test_data` already exists. `./test_data` is now the default so it has no effect.